.github/workflows/test.yaml

```diff
@@ -19,12 +19,14 @@ jobs:
             
       - run: flutter --version
       - run: flutter pub get
+      - run: flutter pub get --directory=example/viam_example_app
 
+      # Verify formatting in all of example, test & lib except lib/gen/
       - name: Verify formatting
-        run: dart format --output=none .
+        run: dart format --set-exit-if-changed example test lib/src/components lib/src/di lib/src/domain lib/src/media lib/src/proto lib/src/resource lib/src/robot lib/src/rpc
```

Comment:
I wish there was a prettier way to do this. ☹️  perhaps we move where `gen/` is located? :thinking: